packages/Icon/README.md

```diff
@@ -18,6 +18,10 @@ npm install @paprika/icon
 
 ## Usage
 
+```
+IMPORTANT: PAPRIKA MAKES USE OF SOME PROPRIETARY ICONS THAT REQUIRE A LICENCE IN ORDER TO BE CONSUMED. PLEASE REPLACE THOSE PROPRIETARY ICONS OUTSIDE OF THE HIGHBOND PLATFORM.
```

Comment:
Should we say which ones? Or maybe just `PLEASE DO NOT CONSUME THESE ICONS DIRECTLY.` for the second sentence?

Reply:
Yes let's add that phrase after it and we should be ok.